impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -99,6 +99,16 @@ struct TexImage2DData {
         external_format = GL_ALPHA;
         type = GL_UNSIGNED_BYTE;
         break;
+      case PixelFormat::kR8UNormInt:
+        internal_format = GL_RED;
+        external_format = GL_RED;
+        type = GL_UNSIGNED_BYTE;
+        break;
+      case PixelFormat::kR8G8UNormInt:
+        internal_format = GL_RG;
```

Comment:
I don't believe these are part of OpenGL ES 2.0 are they?

Reply:
Good catch! I removed these and add a TODO comment with issue id in the test.